x-pack/plugins/apm/public/components/app/RumDashboard/PageViewsTrend/index.tsx

```diff
@@ -41,11 +41,11 @@ export function PageViewsTrend() {
       }
       return Promise.resolve(undefined);
     },
-    [end, start, serviceName, uiFilters, breakdowns]
+    [end, start, serviceName, uiFilters, breakdown]
   );
 
-  const onBreakdownChange = (values: BreakdownItem[]) => {
-    setBreakdowns(values);
+  const onBreakdownChange = (value: BreakdownItem | null) => {
```

Comment:
Do you need to define this function? Couldn't you just pass `onBreakdownChange={setBreakdown}` into the `BreakdownFilter`?

Reply:
done